Workspace id migration (#106)
terraform-providers/terraform-provider-tfe: website/docs/r/variable.html.markdown

  of HCL code. Has no effect for environment variables. Defaults to `false`.
* `sensitive` - (Optional) Whether the value is sensitive. If true then the
  variable is written once and not visible thereafter. Defaults to `false`.

> * `workspace_id` - (Required) The workspace that owns the variable, specified as the workspace's external id (`ws-<RANDOM STRING>`).

Reviewer: ```suggestion
* `workspace_id` - (Required) ID of the workspace that owns the variable.
```
